organizations/ajax_htmldata.php

```diff
@@ -923,6 +923,7 @@ function generateDowntimeHTML($downtime) {
 		}else{
 			$thisPageNum = count($organizationArray) + $pageStart - 1;
 			echo "<span style='font-weight:bold;'>"._("Displaying ") . $pageStart . _(" to ") . $thisPageNum . _(" of ") . $totalRecords . _(" Organization Records")."</span><br />";
+			echo "</span><span style='float:right;width:34px;'><a href='javascript:void(0);'><img src='images/xls.gif' id='export'></a></span>";
```

Comment:
This has a couple accessibility issues:

- there's no label or alt text
- it can't be activated via keyboard navigation; making the element a button is one way of fixing that